sources/account_abstraction/erc4337/base/account_abstraction_erc4337_base_sources.yml

```diff
@@ -1,4 +1,4 @@
-version: 2
+version: 3
```

Comment:
was this update intentional? looks like version isn't required at all anymore, after we upgraded our dbt version
https://docs.getdbt.com/reference/project-configs/version#faqs